packages/client/hmi-client/src/services/model.ts

```diff
@@ -1,25 +1,36 @@
-import API from '@/api/api';
+import API, { getProjectIdFromUrl } from '@/api/api';
 import { useProjects } from '@/composables/project';
 import * as EventService from '@/services/event';
-import type { Initial, InterventionPolicy, Model, ModelConfiguration, ModelParameter } from '@/types/Types';
+import type {
+	Initial,
+	InterventionPolicy,
+	Model,
+	ModelConfiguration,
+	ModelParameter,
+	ProjectAsset
+} from '@/types/Types';
 import { Artifact, EventType } from '@/types/Types';
 import { AMRSchemaNames } from '@/types/common';
 import { fileToJson } from '@/utils/file';
 import { isEmpty } from 'lodash';
 import type { MMT } from '@/model-representation/mira/mira-common';
+import { activeProjectId } from '@/composables/activeProject';
 import { Ref } from 'vue';
 
-export async function createModel(model: Model): Promise<Model | null> {
+export async function createModel(model: Model): Promise<ProjectAsset | null> {
+	const projectId = activeProjectId.value || getProjectIdFromUrl();
 	delete model.id;
-	const response = await API.post(`/models`, model);
+	const response = await API.post(`/models?project-id=${projectId}`, model);
```

Comment:
Is this to save a call to the back end to add the asset to the projectAsset table?
Why not changing the payload to have a model and a projectId for that purpose?